bitcoin/src/blockdata/script.rs

```diff
@@ -139,13 +139,19 @@ pub struct Script([u8]);
 ///
 /// [deref coercions]: https://doc.rust-lang.org/std/ops/trait.Deref.html#more-on-deref-coercion
 #[derive(Default, Debug, Clone, PartialOrd, Ord, PartialEq, Eq, Hash)]
-pub struct ScriptBuf(Vec<u8>);
+pub struct ScriptBuf<T = Generic>(PhantomData<T>, Vec<u8>);
+
+/// Marker for a generic [`ScriptBuf`].
+pub type Generic = ();
```

Comment:
I assume this exists temporarily to decrease the size of the PR and should be eventually removed?